frontend/src/components/dashboard/customHomePage/CustomHomePage.js

```diff
@@ -41,9 +39,12 @@ const propTypes = {
 };
 
 export class CustomHomePage extends Component {
+  state = { isLoaded: false };
+
   componentDidMount() {
-    this.props.fetchSubscribedMinesByUser();
-    this.props.fetchInspectors();
+    this.props.fetchSubscribedMinesByUser().then(() => {
+      this.setState({ isLoaded: true });
+    });
```

Comment:
We didn't need to call `this.props.fetchInspectors();` at all?

Reply:
no, it should have been removed when variances was removed from this view.